### DISCUSSION TITLE
posts/2023/01/making-habits/

### DISCUSSION THREAD

**Discussion by giscus** [naoki-urabe/my-blog#62] [General]
# posts/2023/01/making-habits/

2023年を迎えて新たに習慣づけを行う必要性があるなあと感じたため、「続ける」習慣という本を読んでみました。
今までの人生で何度も勉強や生活習慣に関することで習慣化を試みてきましたが、結局身につかないということを繰り返していました。
最近でいうと、去年始めたこのブログが、11月はまあまあ続いていたのですが、12月にパタっと更新途絶えさせてしまいました。
こうした反省もあり、再度習慣作りの仕組みについて学び、2023年こそはいろいろ習慣化していきたいと思います。

https://nao-eng.com/posts/2023/01/making-habits/

**Comment by EG-yamagami** [comment]
読んでみます！
